app/controllers/admin/users_controller.rb

```diff
@@ -17,7 +17,16 @@
 
 class Admin::UsersController < ApplicationController
   before_action { |controller|
-    controller.send :authenticate_actions_for_admin, 
+    
+  # obtain course (if any)
```

Comment:
When do you need the course information?

Reply:
This is to determine whether the user has any course membership. There is an existing option in SSID whereby admins can delete course membership of an user. In such cases, the user will not have any modules assigned to him.